.github/workflows/helm-release.yaml

```diff
@@ -58,3 +58,20 @@ jobs:
           token: "${{ secrets.GITHUB_TOKEN }}"
           linting: off
           charts_dir: charts-tmp
+
+      - name: Login to GitHub Container Registry
+        run: |
+          helm registry login --username ${GITHUB_ACTOR} --password ${{ secrets.GITHUB_TOKEN }} ghcr.io
+        env:
+          HELM_EXPERIMENTAL_OCI: 1
+      
+      - name: Publish OCI Charts
+        run: |
+          for dir in `find charts-tmp -maxdepth 1 -mindepth 1 -type d -print`; do
+            chart=${dir##*/}
+            echo "Found chart: ${chart}"
+            helm package charts-tmp/${chart} --destination dist
+            helm push dist/${chart}-*.tgz oci://ghcr.io/${GITHUB_REPOSITORY}
+          done
+        env:
+          HELM_EXPERIMENTAL_OCI: 1
```

Comment:
This is also unnecessary with Helm v3.8.0.  I deploy Helm charts as OCI objects to Harbor registry using Helm v3.8.0 and don't need this environment variable.

Reply:
So true, thanks